src/components/StandaloneDirectorChange/CODDate.vue

```diff
@@ -67,7 +67,7 @@ import { DateMixin } from '@/mixins'
 export default class CodDate extends Mixins(DateMixin) {
   // Prop passed into this component.
   @Prop({ default: '' })
-  private initialCODDate: string
+  readonly initialCODDate: string
```

Comment:
`Cod` seems to be used in other places.  do we want to do that here too?

Reply:
OK, I changed variables with "COD" to "Cod" (and same for COA).